docs/security.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight:: javascript
+
+.. _security-reference:
+
+********
+Security
+********
+
+Linz comes with a number of security features out of the box including:
+
+- CSRF Protection
+
+CSRF Protection
+----------------
+
+CSRF protection helps prevent unauthorized commands that are transmitted from a user that the web application trusts.
+
+You can read more about CSRF here https://en.wikipedia.org/wiki/Cross-site_request_forgery.
```

Comment:
Could you please update the link here to https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF), as they're seen as the industry standard when it comes to security.

Reply:
Could you please replace `more about CSRF here` with `more about CSRF on OWASP` and make the link text _CSRF on OWASP_ using the RST external link syntax... http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets